hdbt.theme

```diff
@@ -1346,8 +1353,15 @@ function hdbt_preprocess_tpr_service(&$variables): void {
   if (!isset($variables['entity']) || !$variables['entity'] instanceof Service) {
     return;
   }
+  // Add service publish state to variable.
+  $variables['service_published'] = TRUE;
```

Comment:
This variable could be more generic. Like `$variables['content_entity_published']`

Reply:
Made it generic 🦖